rancher/distros-test-framework
# Move documentation to its own folder and separate ares of responsibility

#### Proposed Changes ####
Have docs live in their own folder and keep main README simple with links to the respective docs.



#### Types of Changes ####


documentation changes

#### Testing ####

N/A

Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.


2. Verify code lint; we should not have errors.


3. Update the documentation if needed.


4. Update makefile and docker run if adding new tests.


5. Run your tests at least 4 times with all configurations needed and possible.


6. If needed test with different os types.


#### Linked Issues ####


https://github.com/rancher/qa-tasks/issues/966

#### Further Comments ####
